docs/download.md

```diff
@@ -65,10 +65,8 @@ for your hardware's minimum driver version.
 * Download the [RAPIDS
   Accelerator for Apache Spark 23.08.1 jar](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/23.08.1/rapids-4-spark_2.12-23.08.1.jar)
 
-This package is built against CUDA 11.8, all CUDA 11.x and 12.x versions are supported through [CUDA forward
-compatibility](https://docs.nvidia.com/deploy/cuda-compatibility/index.html). It is tested
-on V100, T4, A10, A100, L4 and H100 GPUs with CUDA 11.8-12.0.  For those using other types of GPUs 
-which do not have CUDA forward compatibility (for example, GeForce), CUDA 11.8 or later is required.
+This package is built against CUDA 11.8. It is tested on V100, T4, A10, A100, L4 and H100 GPUs with 
+CUDA 11.8-12.0.
```

Comment:
Nit: This looks like a weird single version number.  Would be good to make it clear:
```suggestion
CUDA 11.8 through CUDA 12.0.
```